Bug 1110660 - SIM2's PIN1 code will not pop up if SIM1 is not locked

YangYangSPRD opened the pull request.

Commit:
Bug 1110660 - [Flame] The SIM2's PIN1 code will not pop up when user only insert SIM card to slot 2

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/10435177?v=3" alt="YangYangSPRD" width="50px" height="50px" /> YangYangSPRD](https://github.com/YangYangSPRD) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=bddda8608af7)

jobara committed:
Bug 1086930 - Add a progressbar to the sync button

Added a progress bar inside of the sync button. This is only added to the a11y tree during syncing. It will specify that syncing has started and completed.

rexboy7 committed:
Bug 1122398 - [Stingray][System] Change icon color of context menu in the way Gecko-34 supported

shamenchens committed:
Bug 1123175 - Fix jshint failures in smart-home

shamenchens committed:
Bug 1123176 - Fix jshint failures in smart-settings

shamenchens committed:
Bug 1123173 - Fix jshint failures in tv's browser

shamenchens committed:
Bug 1123174 - Fix jshint failures in dlna-player

shamenchens committed:
Bug 1123191 - Fix jshint failures in tv build script

shamenchens committed:
Bug 1123192 - Fix jshint failures in tv shared

yzen committed:
Merge pull request #26386 from jobara/1086930

Bug 1086930 - Add status region to sync calendar sync button

shamenchens committed:
Merge pull request #27480 from shamenchens/Bug1123173-BrowserJshint

Bug 1123173 - Fix jshint failures in tv's browser, r=johnhu

shamenchens committed:
Merge pull request #27483 from shamenchens/Bug1123174-DlanPlayerJshint

Bug 1123174 - Fix jshint failures in dlna-player, r=rexboy

shamenchens committed:
Merge pull request #27477 from shamenchens/Bug1123175-SmartHomeJshint

Bug 1123175 - Fix jshint failures in smart-home, r=rexboy

shamenchens committed:
Merge pull request #27478 from shamenchens/Bug1123176-SmartSettingsJshint

Bug 1123176 - Fix jshint failures in smart-settings, r=johnhu

shamenchens committed:
Merge pull request #27484 from shamenchens/Bug1123191-TVBuildJshint

Bug 1123191 - Fix jshint failures in tv build script, r=johnhu

shamenchens committed:
Merge pull request #27486 from shamenchens/Bug1123192-TVSharedJshint

Bug 1123192 - Fix jshint failures in tv shared, r=johnhu

timdream committed:
Merge pull request #27505 from Gioyik/bug-108525

Bug 1085250 - Decent icon for the Bluetooth app, r=timdream

rexboy7 committed:
Merge pull request #27452 from rexboy7/1122398-context-menu-icon-change-for-gecko34

Bug 1122398 - [Stingray][System] Change icon color of context menu by svg filter. r=johnhu

YangYangSPRD committed:
Merge branch 'master' of https://github.com/YangYangSPRD/gaia
